src_assets/common/assets/web/apps.html

```diff
@@ -253,6 +296,80 @@ <h1>Applications</h1>
           undo: "",
         });
       },
+      showCoverFinder($event) {
+        this.coverCandidates = [];
+        this.coverSearching = true;
+
+        function getSearchKey(q) {
+          const key = q.toLowerCase().replaceAll(/[^\da-z]+/g, '.');
+          if (key.endsWith('.')) {
+            return key.substring(0, key.length - 1);
+          }
+          return key;
+        }
+
+        function getSearchBucket(key) {
+          let index = key.indexOf('.');
+          if (index < 0) {
+            index = key.length;
+          }
+          const s = key.substring(0, Math.min(2, index));
+          if (!s || s === '.') return '@';
+          return s;
+        }
+
+        function searchCovers(key) {
+          let bucket = getSearchBucket(key);
+          return fetch("https://mariotaku.org/gcdb/buckets/" + bucket + ".json").then(r => {
```

Comment:
I can probably assume this is an instance of https://github.com/mariotaku/gcdb , am I right?

Reply:
> I can probably assume this is an instance of https://github.com/mariotaku/gcdb , am I right?

I believe that is a correct assumption.

These two items appear to be equal.
https://mariotaku.org/gcdb/buckets/cc.json
https://github.com/mariotaku/gcdb/blob/main/docs/buckets/cc.json

I don't know that the first site is pulling directly from github though. I didn't see any call to github when inspecting the site. Maybe it is cloned locally on the server running the site?

Seems that the buck is the first 2 characters of the game title.

Reply:
The site is GH pages with custom domain. I'd like to ask if I could transfer this repo to host under SunshineStream.

Reply:
You can transfer it... I assume I'll need to setup a workflow to update it on a schedule?

Reply:
Yes. I'll add this part and transfer the repo.

Reply:
Okay, I can add secrets if needed... I assume I'll need a few for igdb api.

Reply:
I'm ready for transfer. Could you grant me permission to create public repo under SunshineStream org?

Reply:
@mariotaku I just imported it (much faster)... https://github.com/SunshineStream/db

https://sunshinestream.github.io/db/

Reply:
@ReenigneArcher Thanks. I'll submit to that repo for updates if needed.